src/cli/version.rs

```diff
@@ -57,6 +61,36 @@ fn show_version(out: &mut Output) {
     rtxprintln!(out, "{}", *VERSION);
 }
 
+fn show_latest() {
+    if let Some(latest) = check_for_new_version() {
+        warn!("rtx version {} available", latest)
+    }
+}
+
+pub fn check_for_new_version() -> Option<String> {
+    let current = env!("CARGO_PKG_VERSION").to_string();
+    let current = Versioning::new(current.as_str()).unwrap();
```

Comment:
```suggestion
    let current = Versioning::new(env!("CARGO_PKG_VERSION")).unwrap();
```

what you're doing here is taking `env!("CARGO_PKG_VERSION")`, converting it from a `&str` into a String, then calling `as_str()` which converts it back again. There's no need to do that conversion.